pkg/controller/resourcequota/resource_quota_controller.go

```diff
@@ -196,7 +216,8 @@ func (rq *ResourceQuotaController) Run(workers int, stopCh <-chan struct{}) {
 	}
 	// the workers that chug through the quota calculation backlog
 	for i := 0; i < workers; i++ {
-		go wait.Until(rq.worker, time.Second, stopCh)
+		go wait.Until(rq.worker(rq.queue), time.Second, stopCh)
```

Comment:
does this break some external contract?  should a worker first look at the priority queue and if empty then fall back to the normal queue?